From c75c12cd281dd564ced87f85ce42b2ced82cc9b3 Mon Sep 17 00:00:00 2001
From: Shilin Victor <chrono.monochrome@gmail.com>
Date: Sun, 17 Sep 2017 12:42:53 +0300
Subject: [PATCH 10/14] Fix cm charging screen

Change-Id: I3a9f7c372985479970437d5f55512eda5d195640
---
 charger/Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/charger/Android.mk b/charger/Android.mk
index 46a6695..a96dedb 100644
--- a/charger/Android.mk
+++ b/charger/Android.mk
@@ -1,3 +1,4 @@
+ifneq ($(WITH_LINEAGE_CHARGER),false)
 LOCAL_PATH := $(call my-dir)
 
 # Set healthd_density to the density bucket of the device.
@@ -70,3 +71,4 @@ include $(BUILD_PHONY_PACKAGE)
 
 _add-charger-image :=
 _img_modules :=
+endif
-- 
2.7.4

