From 711fcb733d3c2e538795865227bb162d65ee3202 Mon Sep 17 00:00:00 2001
From: Victor Shilin <chrono.monochrome@gmail.com>
Date: Sat, 19 Aug 2017 21:08:17 +0300
Subject: [PATCH 1/7] 0001-hardware-libhardware_add-AGPS-USE-PSC.patch.patch

---
 include/hardware/gps.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/hardware/gps.h b/include/hardware/gps.h
index 4e108b3..ff70420 100644
--- a/include/hardware/gps.h
+++ b/include/hardware/gps.h
@@ -481,6 +481,9 @@ typedef struct {
      * might rely in the old (wrong) behavior.
      */
     uint16_t lac;
+#ifdef AGPS_USE_PSC
+    uint16_t psc;
+#endif
     /** Cell id in 2G. Utran Cell id in 3G. Cell Global Id EUTRA in LTE. */
     uint32_t cid;
     /** Tracking Area Code in LTE. */
-- 
2.7.4

