From f2e1f0b9a737198cf81e726e0f261bc976dd5b43 Mon Sep 17 00:00:00 2001
From: Victor Shilin <chrono.monochrome@gmail.com>
Date: Sat, 19 Aug 2017 21:08:17 +0300
Subject: [PATCH 2/7] 0002-hardware-libhardware_Samsung-GPS-header.patch.patch

---
 include/hardware/gps.h | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/include/hardware/gps.h b/include/hardware/gps.h
index ff70420..6e05188 100644
--- a/include/hardware/gps.h
+++ b/include/hardware/gps.h
@@ -325,6 +325,13 @@ typedef uint8_t                         GnssConstellationType;
  */
 #define GPS_GEOFENCING_INTERFACE   "gps_geofencing"
 
+#ifdef STE_SAMSUNG_HARDWARE
+/**
+ * The GPS chipset can use Psc for AGPS.
+ */
+#define AGPS_USE_PSC
+#endif
+
 /**
  * Name of the GPS Measurements interface.
  */
@@ -387,6 +394,10 @@ typedef struct {
     float   elevation;
     /** Azimuth of SV in degrees. */
     float   azimuth;
+#ifdef STE_SAMSUNG_HARDWARE
+    /** Unknown Samsung element. */
+    int     padding;
+#endif
 } GpsSvInfo;
 
 typedef struct {
-- 
2.7.4

